python-sdk/src/astro/sql/operators/transform.py

```diff
@@ -103,7 +103,7 @@ def my_sql_statement(table1: Table, table2: Table, execution_date) -> Table:
 def transform_file(
     file_path: str,
     conn_id: str = "",
-    parameters: Mapping | Iterable | None = None,
+    parameters: dict | None = None,
```

Comment:
I don't consider this a breaking change. WDYT?

_mypy_ was complaining about `Mapping | Iterable`, which is why I changed that.

Reply:
I don't consider this breaking

Reply:
Yeah, not a breaking change